### PR TITLE
Change image of erxes because current image gives 404

### PR DESCRIPTION
**Pull request by SemperDisco**
Changed the image of the erxes opportunity because the current image gives a 404 in the test.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -166,7 +166,7 @@
     "difficulty": "medium",
     "description": "Submit 3 issues and 1 PR to their <a href='https://github.com/erxes/erxes'>repo</a>, get a t-shirt and stickers.",
     "reference": "https://erxes.io/hubspot-alternative-erxes-swag",
-    "image": "https://erxes.io/static/images/t-shirt.jpeg",
+    "image": "https://i.imgur.com/T1G9zWs.png",
```
Comment (vikaspotluri123) [inline data.json:169]:
```suggestion
    "image": "https://i.imgur.com/pE9Ml3v.png",
```

Comment (SemperDisco) [reply data.json]: Thanks you for the review! I changed the url and amended the change.

**Comment by SemperDisco** [comment]
@vikaspotluri123 Could it be that my amend broke the deploy preview?

**Comment by aslafy-z** [comment]
Was due to some connection reset during image download.
Force the build to run again did the trick.

**Comment by aslafy-z** [comment]
Thank you @SemperDisco :100:

**Pull request merged**